src/tokenizer.rs

```diff
@@ -191,6 +191,68 @@ impl<'a> Token<'a> {
             BadUrl(_) | BadString(_) | CloseParenthesis | CloseSquareBracket | CloseCurlyBracket
         )
     }
+
+    pub(crate) fn description(&self) -> String {
+        match self {
+            Ident(name) => format!("A ident token '{}'", name),
+            AtKeyword(value) => format!("The value '{}' does not include the `@` marker", value),
+            Hash(value) => format!("The value '{}' does not include the `#` marker", value),
+            IDHash(value) => format!(
+                "The value '{}' does not include the `#` marker, but has a valid ID selector",
+                value
+            ),
+            QuotedString(value) => format!("The value '{}' does not include the quotes", value),
+            UnquotedUrl(value) => format!(
+                "The value '{}' does not include the `url(` `)` markers.",
```

Comment:
This doesn't look right... `UnquotedUrl` is just `url(string-without-quotes)`, not any kind of error.